dev-lang/nasm/nasm-2.12.02.recipe

```diff
@@ -7,13 +7,11 @@ designed to be simple and easy to understand, similar to Intel's but less \
 complex. It supports all currently known x86 architectural extensions, and \
 has strong support for macros."
 HOMEPAGE="https://www.nasm.us/"
-COPYRIGHT="1996-2001 Simon Tatham, Julian Hall
-	 2002-2016 H. Peter Anvin, Jim Kukunas, Cyrill Gorcunov, Frank B. \
-Kotler"
+COPYRIGHT="Copyright 1996-2016 the NASM Authors - All rights reserved."
```

Comment:
We don't include the word `Copyright` nor the `All rights reserved.` :)
```diff
-COPYRIGHT="Copyright 1996-2016 the NASM Authors - All rights reserved."
+COPYRIGHT="1996-2016 the NASM Authors"
```
Hmm, do we keep the `the` before `NASM Authors` ?
(I mean, would `"1996-2016 NASM Authors"`be better?)

Reply:
yes, i'd go with just "1996-2016 NASM Authors"

Reply:
not resolved